lms/views/lti/deep_linking.py

```diff
@@ -106,24 +106,27 @@ def file_picker_to_form_fields_v13(self):
             name="application_instance"
         ).get_current()
 
-        url = self._get_content_url(self.request)
+        message = {
+            "https://purl.imsglobal.org/spec/lti/claim/deployment_id": application_instance.deployment_id,
+            "https://purl.imsglobal.org/spec/lti/claim/message_type": "LtiDeepLinkingResponse",
+            "https://purl.imsglobal.org/spec/lti/claim/version": "1.3.0",
+            "https://purl.imsglobal.org/spec/lti-dl/claim/content_items": [
+                {"type": "ltiResourceLink", "url": self._get_content_url(self.request)}
+            ],
+        }
+
+        if data := self.request.parsed_params["deep_linking_settings"].get("data"):
+            # From: https://www.imsglobal.org/spec/lti-dl/v2p0#deep-linking-response-message
+            #
+            # The https://purl.imsglobal.org/spec/lti-dl/claim/data value must match the value
+            # of the data property of the https://purl.imsglobal.org/spec/lti-dl/claim/deep_linking_settings
+            # claim from the LtiDeepLinkinkingRequest message.
+            #
+            # This claim is required if present in LtiDeepLinkingRequest message.
+            message["https://purl.imsglobal.org/spec/lti-dl/claim/data"] = data
```

Comment:
```suggestion
        # From: 
        #  https://www.imsglobal.org/spec/lti-dl/v2p0#deep-linking-response-message
        #
        # The https://purl.imsglobal.org/spec/lti-dl/claim/data value must 
        # match the value of the data property of the 
        # https://purl.imsglobal.org/spec/lti-dl/claim/deep_linking_settings
        # claim from the `LtiDeepLinkinkingRequest` message.
        #
        # This claim is required if present in the `LtiDeepLinkingRequest` 
        # message.
        if data := self.request.parsed_params["deep_linking_settings"].get("data"):
            message["https://purl.imsglobal.org/spec/lti-dl/claim/data"] = data
```

Mostly trying to keep the code together here so I don't forget what the line above was by the time I've finished reading the comment. Tiny brain etc.